Jenkinsfile

```diff
@@ -179,8 +179,14 @@ pipeline {
           withCredentials([usernamePassword(credentialsId: ZOWE_ROBOT_TOKEN, usernameVariable: 'USERNAME', passwordVariable: 'TOKEN')]) { script {
             sh "git push --tags https://$TOKEN:x-oauth-basic@github.com/zowe/vscode-extension-for-zowe.git"
 
+            //Grab changelog, convert to unix line endings, get changes under current version, publish release to github with changes in body
+            def releaseVersion = sh(returnStdout: true, script: "echo ${version} | cut -c 2-").trim()
+            sh "npm install ssp-dos2unix"
+            sh "npm run d2u"
+            def releaseChanges = sh(returnStdout: true, script: "awk -v ver=${releaseVersion} '/## / {if (p) { exit }; if (\$2 ~ ver) { p=1; next} } p && NF' CHANGELOG.md | tr \\\" \\` | sed -z 's/\\n/\\\\n/g'").trim()
```

Comment:
Looks complicated 🤣

Reply:
It's just a bit `awk`ward to read.
The `tr` and `sed` aren't too bad though.

And yes, @tjohnsonBCM, I used your joke. :)